repair/featurize/constraintfeat.py

```diff
@@ -38,28 +58,23 @@ def gen_feat_tensor(violations, total_vars, classes):
 
 
 class ConstraintFeat(Featurizer):
```

Comment:
Would be nice to have a docstring for each featurizer describing either the intuition behind using these features, or what feature these featurizers are generating.